### PR TITLE
add scroll, change dropdown appearing logic

### PR DESCRIPTION
**Pull request by MariaPonomarenko38**

### CODE REVIEW

File: sports_hub/sports_hub/Views/AdminPage/Articles.cshtml [hunk 1]
```diff
@@ -322,4 +337,12 @@
             }
        });
     }
+
+    //$("main-content").scroll(function () {
+    //    if ($("main-content").scrollTop() == $(document).height() - $("main-content").height()) {
+    //        alert("hello");
+    //    }
+    //});
```
Comment (XashDM) [inline sports_hub/sports_hub/Views/AdminPage/Articles.cshtml:345]: remove if not needed

File: sports_hub/sports_hub/sports_hub.csproj [hunk 2]
```diff
@@ -18,6 +18,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="PagedList.Mvc" Version="4.5.0" />
```
Comment (XashDM) [inline sports_hub/sports_hub/sports_hub.csproj:21]: Is this needed?

**Review by XashDM — APPROVED** [review]
Please resolve comments and answer questions

File: sports_hub/sports_hub/sports_hub.csproj (hunk 2 above)
Comment (MariaPonomarenko38) [reply sports_hub/sports_hub/sports_hub.csproj:21]: > Is this needed?

in this pull request - no

**Review by XashDM — APPROVED** [review]
Looks ok

**Pull request merged**